tslearn/shapelets.py

```diff
@@ -893,12 +958,17 @@ def __init__(self, n_shapelets_per_size=None,
                              total_lengths=total_lengths,
                              random_state=random_state)
         self.learning_rate = learning_rate
+        warnings.warn(
```

Comment:
The old SerializableShapeletModel allowed to specify a learning rate as well. As this is redundant now, as that can be specified within the Optimizer-object, we could perhaps add it to the ShapeletModel until version 0.6 to allow backward compatibility even more?

Reply:
Well, I think I'd prefer that in future versions, there is only one way to set the learning rate (through the optimizer). We had this parameter for `SerializableShapeletModel` because we were unable to clone when a custom optimizer was provided. But we no longer have this problem.